MAP/src/components/Map/util.js

```diff
@@ -19,6 +19,25 @@ export function worldStyle(lockdown_status) {
   return value;
 }
 
+export function worldStyleColor(lockdown_status) {
```

Comment:
"lockdown" is only applicable for DSL 1. I think the variable name might be better called something like `style_selector`